csrc/root_domain_map.cpp

```diff
@@ -229,6 +229,30 @@ std::unordered_map<IterDomain*, IterDomain*> PairwiseRootDomainMap::map(
     return dom_map;
   }
 
+  if (SdpaFwdOp* op = dynamic_cast<SdpaFwdOp*>(consumer_tv_->definition())) {
+    // Consumers: output = [N, H, L, Ev], logsumexp = [N, H, L], cum_seq_q/k =
+    // [N + 1,] Producers: query = [N, H, L, E], key = [N, H, S, E], value = [N,
+    // H, S, Ev]
```

Comment:
Readability suggestion:
```suggestion
    // Producers:
    //   query = [N, H, L, E]
    //   key = [N, H, S, E]
    //   value = [N, H, S, Ev]
    // Consumers:
    //   output = [N, H, L, Ev]
    //   logsumexp = [N, H, L]
    //   cum_seq_q/k = [N + 1]
```